### PR TITLE
Update on client homepage

### PR DESCRIPTION
**Pull request by sayo96825**
update on client homepage with background image

### CODE REVIEW

File: LIMS_IMAGE/web/templates/home.html [hunk 1]
```diff
@@ -21,7 +21,7 @@ <h1>LIMS Home</h1>
         <p><i>You are not logged in</i></p>
         <section id ="main">
           <div class ="container">
-              <img src ="{% static 'image/homepage_image.jpg' %}" alt ="Background image with Laboraroty theme" height = 500px>
+              <img src ="{% static 'image/client-homepage_image.jpg' %}" alt ="Background image with Laboraroty theme" height = 350px>
```
Comment (shepkeira) [inline LIMS_IMAGE/web/templates/home.html:24]: why did this image change?

File: LIMS_IMAGE/web/templates/orders/home_page.html [hunk 2]
```diff
@@ -24,6 +40,8 @@ <h1>Client Home</h1>
     <a href="{% url 'login' %}">Log In</a>
     or
     <a href="{% url 'accounts:registration' %}">Sign Up</a>
+
```
Comment (shepkeira) [inline LIMS_IMAGE/web/templates/orders/home_page.html:43]: Please remove these extra space

**Review by shepkeira — CHANGES_REQUESTED** [review]
There are two images being added (they are the same image), please only add the one being used

**Comment by shepkeira** [comment]
The image looks really blurry, if you are able to find a large size image/better resolution, that would be good.
If you don't have time, no worries

**Comment by sayo96825** [comment]
This image is less blurry.  Please let me know if there is anything else I need to change. 🙏

**Comment by shepkeira** [comment]
@sayo96825 there are some merge conflicts you will need to fix

File: LIMS_IMAGE/web/templates/orders/home_page.html [hunk 3]
```diff
@@ -1,10 +1,26 @@
...
+    <link rel="stylesheet" href="{% static 'css/home.css' %}"/> 
+    {% endblock %}
+  </head>
 <body>
   {% block content%}
   <div class="container">
     <h1>Client Home</h1>
+    <section id ="main">
+      <div class ="container my-5">
+          <img src ="{% static 'image/client_page_image.jpg' %}" alt ="Background image with Laboraroty theme" height = 400px>
```
Comment (shepkeira) [inline LIMS_IMAGE/web/templates/orders/home_page.html:21]: this isn't the url of the image you kep

File: LIMS_IMAGE/web/templates/orders/home_page.html [hunk 4]
```diff
@@ -23,7 +39,7 @@ <h1>Client Home</h1>
     <p>You are not logged in</p>
     <a href="{% url 'login' %}">Log In</a>
     or
-    <a href="{% url 'accounts:registration' %}">Sign Up</a>
+    <a href="{% url 'accounts:registration' %}">Sign Up</a>
```
Comment (shepkeira) [inline LIMS_IMAGE/web/templates/orders/home_page.html:42]: while you are at it, please remove the indeed extra space at the end of this line

**Review by shepkeira — CHANGES_REQUESTED** [review]
This image is much better, thank you

File: LIMS_IMAGE/web/templates/orders/home_page.html (hunk 3 above)
Comment (shepkeira) [reply LIMS_IMAGE/web/templates/orders/home_page.html:21]: 'image/client-homepage.jpg'

**Comment by shepkeira** [comment]
@sayo96825 , I fixed your merge changes. Please take a look at them before you merge this PR

**Review by shepkeira — APPROVED** [review]

**Comment by sayo96825** [comment]
> @sayo96825 , I fixed your merge changes. Please take a look at them before you merge this PR

Thank you.

**Pull request merged**